aws-cdk/build.gradle

```diff
@@ -0,0 +1,15 @@
+plugins {
+    id("io.micronaut.build.internal.module")
+}
+
+dependencies {
+    api libs.aws.cdk.lib
+    api("io.micronaut.starter:micronaut-starter-api:$micronautVersion")
```

Comment:
`api` or `implementation`?

Reply:
It is `api` intentionally. So that uses can provide both enums `BuildTool` and `ApplicatonType`.